frontend/src/views/applications/ServiceDeliveryView.vue

```diff
@@ -153,6 +155,12 @@ export default {
           await ApplicationService.updateApplication(this.$route.params.applicationGuid, payload)
           await this.getApplication(this.$route.params.applicationGuid)
         }
+        if (this.changedLicences.length > 0) {
+          for (const licence of this.changedLicences) {
```

Comment:
good catch - I have had issues before using forEach with await - but I think because this is simple and it doesn't matter what license gets sent first, that it works okay. good to know :)

Reply:
Closing but see Viet's latest comments.